presto-main/src/main/java/com/facebook/presto/operator/ExplainAnalyzeOperator.java

```diff
@@ -146,7 +146,7 @@ public Page getOutput()
             return null;
         }
 
-        String plan = textDistributedPlan(queryInfo.getOutputStage().get().getSubStages(), metadata, operatorContext.getSession());
```

Comment:
It's not strictly needed - it's a readability and contract improvement: that method seems to require a very specific list of `subStages` - the one from `outputStage`. Previously it was not clear which exactly `stageInfos` it needed. See [here](https://github.com/prestodb/presto/pull/7442/commits/c1254b4a969f63e7cfad106abde142cd4d773ca6#diff-11a661ee627130692b6f5c4ab02b1354R183) for the signature change.